### PR TITLE
Updated Readme

### PR DESCRIPTION
**Pull request by brianmusslewhite**
I updated the readme to be a little more engaging. Let me know your thoughts.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,13 @@
+# ArduPilot Wiki Editing Guide
 
-# ArduPilot Wiki Sources
+Visit our main wiki page at [ArduPilot](https://ardupilot.org/ardupilot).
 
-## ArduPilot Wiki Editing Guide
+For comprehensive editing guidelines, please refer to our [Editing Guide](https://ardupilot.org/dev/docs/common-wiki_editing_guide.html).
```
Comment (Hwurzburg) [inline README.md:5]:
```suggestion
For comprehensive editing guidelines, please refer to our [Editing Guide](https://ardupilot.org/dev/docs/common-wiki_editing_guide.html). Most wiki pages also allow proposing changes, using rudimentary direct editing of the page by clicking the top right link: "Edit on GitHub". 
```

**Review by Hwurzburg — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Hwurzburg) [reply README.md:5]: please squash to one commit after making the changes...thanks

**Comment by brianmusslewhite** [comment]
@Hwurzburg updated and squashed

**Comment by Hwurzburg** [comment]
LGTM...thanks

**Pull request merged**